### ISSUE TITLE
webify download is not executable

### ISSUE THREAD

**Issue by btilford** [caiguanhao/grunt-ziti#2]
At least on linux this causes the spawn to fail with,
<pre>
<code>
Generating web fonts... Fatal error: spawn EACCES
Error: spawn EACCES
    at errnoException (child_process.js:1000:11)
    at Process.ChildProcess._handle.onexit (child_process.js:791:34)
</code>
</pre>

Running `chmod +x node_modules/grunt-ziti/vendor/webify` fixes the issue.

**Comment by caiguanhao** [comment]
I have tried running the webify task in a Node.JS docker container and it worked. This https://github.com/caiguanhao/grunt-ziti/blob/3ff0ae1f2b8eb7df4bf86fb2b2e63878de8f6d0a/tasks/ziti.js#L44 shows the webify executable is set to 0755 when the download completes. You can try reinstalling the package and try again. If the problem still exists, please tell me your system distro and node.js, npm version.

**Comment by btilford** [comment]
I haven't been able to reproduce this consistently. The download function is quite hard to follow/debug.
